iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -128,9 +128,10 @@ public enum ErrorCode
         JobQuotaExceeded = 403003,
 
         /// <summary>
-        /// The underlying cause is that the number of messages enqueued for the device exceeds the queue limit (50).
-        /// The most likely reason that you're running into this limit is because you're using HTTPS to receive the message,
-        /// which leads to continuous polling using ReceiveAsync, resulting in IoT hub throttling the request.
+        /// The underlying cause is that the number of cloud-to-device messages enqueued for the device exceeds the queue limit (50).
+        /// You will need to receive and complete/reject the messages from the device-side before you can enqueue any additional messages.
+        /// If you want to discard the currently enqueued messages,
+        /// you can <see href="https://github.com/Azure/azure-iot-sdk-csharp/blob/c76a64de272da986b6840251f482249e094a725c/iothub/service/src/ServiceClient.cs#L211">purge your device message queue</see>.
```

Comment:
Could you link the method within our codebase via cref rather than through github?

Reply:
sure